packaging/cockpit-starter-kit.spec.in

```diff
@@ -8,7 +8,10 @@ Source0: https://github.com/cockpit-project/starter-kit/releases/download/%{vers
 Source1: https://github.com/cockpit-project/starter-kit/releases/download/%{version}/%{name}-node-%{version}.tar.xz
 BuildArch: noarch
 ExclusiveArch: %{nodejs_arches} noarch
+%if 0%{?fedora}
 BuildRequires: nodejs-devel
+%endif
+BuildRequires: nodejs
```

Comment:
Please squash this into the first commit. Projects often cherry-pick commits from starter-kit, and the first commit is broken by itself (as it fails on RHEL).